Gruntfile.js

```diff
@@ -236,7 +236,8 @@ module.exports = function(grunt) {
           buildCSS: false,
           paths: {
             'css-builder': '../build/css-builder',
-            'normalize': '../build/normalize'
+            'normalize': '../build/normalize',
+            'jquery': 'empty:'
```

Comment:
It's not a grunt thing but a requirejs config option: http://requirejs.org/docs/optimization.html#empty
But I have no idea why it solves the problem.